app/helpers/admin_helper.rb

```diff
@@ -1,13 +1,24 @@
 require 'csv'
+require 'date'
+require 'time'
+
 
 module AdminHelper
 
   def typeMissingMessage
     "Creation failed. Please input either email, phone number, or both."
   end
 
-  def errorMessage
-    "An error has occurred saving your user. Please contact the administrator."
+  def eventSuccessMessage
+    "Your event was successful!"
```

Comment:
We're gonna update these to use language from a .yaml file in another story. Good with me though for this push! Like how you did these in a helper.